packages/connector-oidc/README.md

```diff
@@ -148,11 +76,11 @@ If 'id_token' is not presented in `responseType`, then `tokenEndpoint` is requir
 
 | Name                                | Type                                | Required  |
 |-------------------------------------|-------------------------------------|-----------|
-| oidcFlowType                        | enum                                | True      |
 | scope                               | string                              | True      |
 | clientId                            | string                              | True      |
 | clientSecret                        | string                              | True      |
 | authorizationEndpoint               | string                              | True      |
+| tokenEndpoint                       | string                              | True      |
```

Comment:
So this parameter is missing in the previous code?

Reply:
Missed in previous README --  it should be optional in the past and will be REQUIRED from now on since we only support authorization code flow now.